cmd/traefik/traefik.go

```diff
@@ -531,14 +531,23 @@ func registerMetricClients(metricsConfig *types.Metrics) []metrics.Registry {
 
 	if metricsConfig.InfluxDB2 != nil {
 		ctx := log.With(context.Background(), log.Str(log.MetricsProviderName, "influxdb2"))
-		influxDB2Register := metrics.RegisterInfluxDB2(ctx, metricsConfig.InfluxDB2)
-		if influxDB2Register != nil {
-			registries = append(registries, influxDB2Register)
+		influxDB2Registry := metrics.RegisterInfluxDB2(ctx, metricsConfig.InfluxDB2)
+		if influxDB2Registry != nil {
+			registries = append(registries, influxDB2Registry)
 			log.FromContext(ctx).Debugf("Configured InfluxDB v2 metrics: pushing to %s (%s org/%s bucket) once every %s",
 				metricsConfig.InfluxDB2.Address, metricsConfig.InfluxDB2.Org, metricsConfig.InfluxDB2.Bucket, metricsConfig.InfluxDB2.PushInterval)
 		}
 	}
 
+	if metricsConfig.OpenTelemetry != nil {
+		ctx := log.With(context.Background(), log.Str(log.MetricsProviderName, "openTelemetry"))
+		openTelemetryRegistry := metrics.RegisterOpenTelemetry(ctx, metricsConfig.OpenTelemetry)
+		if openTelemetryRegistry != nil {
+			registries = append(registries, openTelemetryRegistry)
+			log.FromContext(ctx).Debugf("Configured OpenTelemetry metrics: pushing to %q", metricsConfig.OpenTelemetry.Address)
```

Comment:
we indeed do not have any org or bucket to add to the log message, but we do have a pushInterval though, no?